templates-generator/angular-config.js

```diff
@@ -0,0 +1,260 @@
+                    after: '"items": []'
+                },
+                {
+                    before: '"assetsBasePath": "../',
+                    after: '"assetsBasePath": "<%= path %>'
+                }
+            ]
+        },
+        {
+            glob: 'src/themes/**/*.*',
+            definitions: [
+                {
+                    before: /"items":\s+\[[^]*?\]/,
+                    after: '"items": []'
+                },
+                {
+                    before: /"assetsBasePath": ".*?"/,
+                    after: '"assetsBasePath": "<%= path %>../../node_modules/devextreme/dist/css/"'
+                }
+            ]
+        },
+        {
+            glob: 'devextreme.json',
+            definitions: [
+                {
+                    before: /"applicationEngine": ".*?"/,
+                    after: '"applicationEngine": "<%= engine %>"'
+                },
+                {
+                    before: /"outputFile": "src/g,
+                    after: '"outputFile": "<%= sourcePath %>'
+                },
+                {
+                    before: /"inputFile": "src/g,
+                    after: '"inputFile": "<%= sourcePath %>'
+                }
+            ]
+        },
+        {
+            glob: '{src/app/shared/services/app-info.service.ts,e2e/src/app.e2e-spec.ts}',
+            definitions: [
+                {
+                    before: 'My App',
+                    after: '<%= title %>'
+                }
+            ]
+        },
+        {
+            glob:'src/app/pages/home/home.component.html',
+            definitions: [
+                {
+                    before: 'My App',
+                    after: '<%= project %>'
+                }
+            ]
+        },
+        {
+            glob: 'components/side-navigation-menu/side-navigation-menu.component.ts',
+            definitions: [
+                {
+                    before: /@ViewChild\([^\)]*?/,
+                    after: '@ViewChild(DxTreeViewComponent, { static: true })'
+                }
+            ]
+        },
+        {
+            glob: 'src/app/app-navigation.ts',
+            definitions: [
+                {
+                    before: /export const navigation =\s+\[[\s\S]*?\];/,
+                    after: 'export const navigation = [];'
+                }
+            ]
+        },
+        {
+            glob: 'src/app/shared/components/side-navigation-menu/side-navigation-menu.component.ts',
+            definitions: [
+                {
+                    before: /@ViewChild\(DxTreeViewComponent.*?\)/,
+                    after: '@ViewChild(DxTreeViewComponent<% if(requireStaticFlag) { %>, { static: true }<% } %>)'
+                }
+            ]
+        },
+        {
+            glob: 'src/app/app-routing.module.ts',
+            definitions: [
+                {
+                    before: /import { HomeComponent } from '.\/pages\/home\/home.component';[\r\n]/,
```

Comment:
Why you are using regular expression here?

Reply:
Regular Expression will work with different line endings

Reply:
I replaced expression